### PR TITLE
fix: fix deps

### PR DESCRIPTION
**Pull request by codeudan**
1.move @swc/core in dependencies  to devDependencies. if  "@swc/core" in "dependencies", it will be installed twice when user  install it  , it should in devDependencies
2.@swc/core in peerDependencies should a loose version

### CODE REVIEW

**Comment by codeudan** [comment]
@kdy1  hi~, can you help to review this pr?😀

File: package.json [hunk 1]
```diff
@@ -8,14 +8,14 @@
     "license": "MIT",
     "private": false,
     "devDependencies": {
-        "webpack": "^4.28.3"
+        "webpack": "^4.28.3",
+        "@swc/core": "^1.2.52"
     },
     "peerDependencies": {
-        "@swc/core": "^1.2.52",
+        "@swc/core": "^1.2.0",
```
Comment (kdy1) [inline package.json:15]: Please use `^1.2.52`.
`@swc/cli` pass a hidden argument to `@swc/core`.

**Review by kdy1 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (codeudan) [reply package.json]: @kdy1 Done.

**Review by kdy1 — APPROVED** [review]
Thanks!

**Pull request merged**